admin/requirements.py

```diff
@@ -14,7 +14,7 @@
 from twisted.python.usage import Options, UsageError
 from twisted.python.filepath import FilePath
 
-MANYLINUX_IMAGE = "quay.io/pypa/manylinux1_x86_64:latest"
+ALPINE_IMAGE = "alpine:latest"
```

Comment:
:+1: I agree but I'll leave it for now....to save having to re-run all the tests :frowning: